### ISSUE TITLE
How to install Gesturefy on Microsoft Edge?

### ISSUE THREAD

**Issue by bxtom** [Robbendebiene/Gesturefy#600]
### Describe your matter in detail
Millions of Firefox users are moving to other browsers. Many of them are choosing Edge but there is no Gesturefy among Edge extensions. How to install it?

### Screenshots or further assistance
no screenshots

### Your System
- Operating system: W10
- Firefox version: latest
- Gesturefy version: latest

**Comment by Robbendebiene** [comment]
Since Microsoft Edge uses Chromium under the hood the same reasons apply that I have given for Chrome and similar browsers here:

- https://github.com/Robbendebiene/Gesturefy/issues/458
- https://github.com/Robbendebiene/Gesturefy/issues/218
- https://github.com/Robbendebiene/Gesturefy/issues/380

TLDR:
Other browsers lack multiple important APIs that Gesturefy relies on which are only supported by Firefox. Therefore porting is not possible or at least requires a lot of work and sill comes with a loss in functionality.
